.github/workflows/kernel-apply-patch.sh

```diff
@@ -36,7 +36,17 @@ done
 
 popd >/dev/null || exit
 
-URL="https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tag/?h=v${VERSION_NEW}"
+if ! curl -sA 'Chrome' -L 'http://www.google.com/search?hl=en&q=site%3Alwn.net+linux+'"${VERSION_NEW}" -o search.html; then
```

Comment:
```suggestion
if ! curl -sfA 'Chrome' -L 'http://www.google.com/search?hl=en&q=site%3Alwn.net+linux+'"${VERSION_NEW}" -o search.html; then
```
For the return code to be set on `404` you also need `-f`.

Reply:
Sure. Curl can fail for multitude of reasons, which is why I've put it into an `if`. `404` will result in grep not finding anything.